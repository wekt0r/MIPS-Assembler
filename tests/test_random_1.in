srl $5, $22, 14
div $v0, $k1
xori $29, $t4, 22814
sllv $sp, $3, $18
div $t0, $25
andi $31, $t1, 11238
or $19, $ra, $t8
sll $s4, $7, 30
srl $5, $30, 20
sltu $a1, $fp, $19
addu $fp, $29, $23
srlv $4, $17, $31
sltu $27, $s5, $t3
ori $4, $t1, 18418
srlv $t1, $7, $at
slti $s0, $s5, 304
mflo $s2
addu $a3, $26, $ra
multu $30, $16
xori $s0, $s4, 12704
mfhi $s0
xori $21, $24, 13074
addi $at, $s6, 12287
xor $0, $t0, $t6
slt $s5, $30, $k1
divu $s1, $12
mfhi $0
sllv $fp, $t1, $10
sub $2, $8, $s0
sra $t3, $11, 3
andi $s6, $21, 29094
sltu $25, $17, $a2
lui $0, 22121
sltu $7, $8, $22
slt $t8, $k0, $ra
sllv $s6, $4, $23
addu $s3, $ra, $29
mflo $zero
srl $7, $2, 11
slti $28, $t6, 8653
srlv $v0, $t7, $17
xor $gp, $v1, $22
slt $27, $8, $k1
sll $t1, $at, 19
sllv $8, $a1, $k1
mult $t1, $sp
subu $v0, $7, $s1
andi $25, $9, 11451
slt $t0, $ra, $10
sll $15, $s3, 7
srl $s6, $t5, 23
addi $24, $23, 4134
xori $13, $28, 11821
mfhi $s6
or $zero, $7, $s0
srav $6, $sp, $14
slti $at, $19, 16529
multu $s6, $s4
sra $v0, $v1, 29
slt $a0, $4, $17
addu $12, $28, $t7
andi $11, $5, 22888
or $t8, $5, $s4
add $at, $t6, $25
srl $10, $29, 2
sllv $zero, $17, $t4
sub $7, $s3, $20
or $v1, $t4, $18
sltiu $26, $k0, 12478
and $zero, $s5, $s6
slt $13, $t9, $t8
srlv $15, $a3, $t9
sltiu $26, $sp, 11989
srl $gp, $a0, 13
mfhi $t0
or $31, $k1, $t0
mflo $15
xor $7, $ra, $13
mult $5, $13
subu $s4, $a1, $t9
srlv $5, $s3, $24
sra $8, $25, 24
mfhi $1
srl $a3, $t1, 15
ori $12, $12, 29269
or $20, $3, $15
mult $22, $t5
srlv $2, $t4, $s5
srav $zero, $10, $fp
srav $s6, $19, $20
sll $t2, $t0, 30
srav $2, $a3, $9
slt $t8, $30, $17
xori $t6, $s6, 22953
sltiu $t6, $k0, 26601
mfhi $26
mfhi $a1
and $0, $k0, $19
mfhi $9
nor $t9, $10, $zero
divu $t1, $6
andi $5, $15, 3840
div $zero, $12
srav $22, $a3, $s3
sllv $27, $v1, $10
multu $s6, $t2
addi $13, $12, 6554
srav $t9, $t7, $0
add $17, $24, $t7
sltu $26, $ra, $9
srav $19, $20, $a3
srl $3, $s0, 28
sltiu $11, $s0, 25170
slt $4, $a1, $zero
mflo $26
ori $12, $1, 17938
divu $29, $22
srlv $ra, $gp, $t6
srl $at, $s0, 24
subu $s0, $24, $t0
addiu $28, $17, 11186
mfhi $4
and $t7, $t3, $t8
srlv $s2, $3, $6
mflo $a3
div $14, $zero
sub $t3, $15, $9
and $v0, $t9, $18
sltu $t4, $t6, $30
add $17, $31, $t0
sltiu $k0, $30, 19152
srav $t7, $t2, $12
slti $0, $a1, 14790
sllv $8, $v1, $29
srav $t3, $a1, $t9
lui $ra, 19727
sltu $t5, $t9, $9
multu $t2, $31
ori $15, $0, 14550
and $s7, $16, $31
mflo $s6
lui $10, 2163
multu $22, $a2
lui $0, 19027
slti $t4, $s5, 29885
addu $s1, $10, $19
srl $t4, $t0, 22
andi $a1, $s2, 8163
sll $20, $18, 3
sll $11, $a2, 20
mflo $a3
sltu $18, $t9, $27
or $20, $2, $t6
