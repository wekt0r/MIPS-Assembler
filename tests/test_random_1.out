.text
00000000	00162B82	srl	$5,$22,14
00000004	005B001A	div	$v0,$k1
00000008	399D591E	xori	$29,$t4,22814
0000000c	0243E804	sllv	$sp,$3,$18
00000010	0119001A	div	$t0,$25
00000014	313F2BE6	andi	$31,$t1,11238
00000018	03F89825	or	$19,$ra,$t8
0000001c	0007A780	sll	$s4,$7,30
00000020	001E2D02	srl	$5,$30,20
00000024	03D3282B	sltu	$a1,$fp,$19
00000028	03B7F021	addu	$fp,$29,$23
0000002c	03F12006	srlv	$4,$17,$31
00000030	02ABD82B	sltu	$27,$s5,$t3
00000034	352447F2	ori	$4,$t1,18418
00000038	00274806	srlv	$t1,$7,$at
0000003c	2AB00130	slti	$s0,$s5,304
00000040	00009012	mflo	$s2
00000044	035F3821	addu	$a3,$26,$ra
00000048	03D00019	multu	$30,$16
0000004c	3A9031A0	xori	$s0,$s4,12704
00000050	00008010	mfhi	$s0
00000054	3B153312	xori	$21,$24,13074
00000058	22C12FFF	addi	$at,$s6,12287
0000005c	010E0026	xor	$0,$t0,$t6
00000060	03DBA82A	slt	$s5,$30,$k1
00000064	022C001B	divu	$s1,$12
00000068	00000010	mfhi	$0
0000006c	0149F004	sllv	$fp,$t1,$10
00000070	01101022	sub	$2,$8,$s0
00000074	000B58C3	sra	$t3,$11,3
00000078	32B671A6	andi	$s6,$21,29094
0000007c	0226C82B	sltu	$25,$17,$a2
00000080	3C005669	lui	$0,22121
00000084	0116382B	sltu	$7,$8,$22
00000088	035FC02A	slt	$t8,$k0,$ra
0000008c	02E4B004	sllv	$s6,$4,$23
00000090	03FD9821	addu	$s3,$ra,$29
00000094	00000012	mflo	$zero
00000098	00023AC2	srl	$7,$2,11
0000009c	29DC21CD	slti	$28,$t6,8653
000000a0	022F1006	srlv	$v0,$t7,$17
000000a4	0076E026	xor	$gp,$v1,$22
000000a8	011BD82A	slt	$27,$8,$k1
000000ac	00014CC0	sll	$t1,$at,19
000000b0	03654004	sllv	$8,$a1,$k1
000000b4	013D0018	mult	$t1,$sp
000000b8	00F11023	subu	$v0,$7,$s1
000000bc	31392CBB	andi	$25,$9,11451
000000c0	03EA402A	slt	$t0,$ra,$10
000000c4	001379C0	sll	$15,$s3,7
000000c8	000DB5C2	srl	$s6,$t5,23
000000cc	22F81026	addi	$24,$23,4134
000000d0	3B8D2E2D	xori	$13,$28,11821
000000d4	0000B010	mfhi	$s6
000000d8	00F00025	or	$zero,$7,$s0
000000dc	01DD3007	srav	$6,$sp,$14
000000e0	2A614091	slti	$at,$19,16529
000000e4	02D40019	multu	$s6,$s4
000000e8	00031743	sra	$v0,$v1,29
000000ec	0091202A	slt	$a0,$4,$17
000000f0	038F6021	addu	$12,$28,$t7
000000f4	30AB5968	andi	$11,$5,22888
000000f8	00B4C025	or	$t8,$5,$s4
000000fc	01D90820	add	$at,$t6,$25
00000100	001D5082	srl	$10,$29,2
00000104	01910004	sllv	$zero,$17,$t4
00000108	02743822	sub	$7,$s3,$20
0000010c	01921825	or	$v1,$t4,$18
00000110	2F5A30BE	sltiu	$26,$k0,12478
00000114	02B60024	and	$zero,$s5,$s6
00000118	0338682A	slt	$13,$t9,$t8
0000011c	03277806	srlv	$15,$a3,$t9
00000120	2FBA2ED5	sltiu	$26,$sp,11989
00000124	0004E342	srl	$gp,$a0,13
00000128	00004010	mfhi	$t0
0000012c	0368F825	or	$31,$k1,$t0
00000130	00007812	mflo	$15
00000134	03ED3826	xor	$7,$ra,$13
00000138	00AD0018	mult	$5,$13
0000013c	00B9A023	subu	$s4,$a1,$t9
00000140	03132806	srlv	$5,$s3,$24
00000144	00194603	sra	$8,$25,24
00000148	00000810	mfhi	$1
0000014c	00093BC2	srl	$a3,$t1,15
00000150	358C7255	ori	$12,$12,29269
00000154	006FA025	or	$20,$3,$15
00000158	02CD0018	mult	$22,$t5
0000015c	02AC1006	srlv	$2,$t4,$s5
00000160	03CA0007	srav	$zero,$10,$fp
00000164	0293B007	srav	$s6,$19,$20
00000168	00085780	sll	$t2,$t0,30
0000016c	01271007	srav	$2,$a3,$9
00000170	03D1C02A	slt	$t8,$30,$17
00000174	3ACE59A9	xori	$t6,$s6,22953
00000178	2F4E67E9	sltiu	$t6,$k0,26601
0000017c	0000D010	mfhi	$26
00000180	00002810	mfhi	$a1
00000184	03530024	and	$0,$k0,$19
00000188	00004810	mfhi	$9
0000018c	0140C827	nor	$t9,$10,$zero
00000190	0126001B	divu	$t1,$6
00000194	31E50F00	andi	$5,$15,3840
00000198	000C001A	div	$zero,$12
0000019c	0267B007	srav	$22,$a3,$s3
000001a0	0143D804	sllv	$27,$v1,$10
000001a4	02CA0019	multu	$s6,$t2
000001a8	218D199A	addi	$13,$12,6554
000001ac	000FC807	srav	$t9,$t7,$0
000001b0	030F8820	add	$17,$24,$t7
000001b4	03E9D02B	sltu	$26,$ra,$9
000001b8	00F49807	srav	$19,$20,$a3
000001bc	00101F02	srl	$3,$s0,28
000001c0	2E0B6252	sltiu	$11,$s0,25170
000001c4	00A0202A	slt	$4,$a1,$zero
000001c8	0000D012	mflo	$26
000001cc	342C4612	ori	$12,$1,17938
000001d0	03B6001B	divu	$29,$22
000001d4	01DCF806	srlv	$ra,$gp,$t6
000001d8	00100E02	srl	$at,$s0,24
000001dc	03088023	subu	$s0,$24,$t0
000001e0	263C2BB2	addiu	$28,$17,11186
000001e4	00002010	mfhi	$4
000001e8	01787824	and	$t7,$t3,$t8
000001ec	00C39006	srlv	$s2,$3,$6
000001f0	00003812	mflo	$a3
000001f4	01C0001A	div	$14,$zero
000001f8	01E95822	sub	$t3,$15,$9
000001fc	03321024	and	$v0,$t9,$18
00000200	01DE602B	sltu	$t4,$t6,$30
00000204	03E88820	add	$17,$31,$t0
00000208	2FDA4AD0	sltiu	$k0,$30,19152
0000020c	018A7807	srav	$t7,$t2,$12
00000210	28A039C6	slti	$0,$a1,14790
00000214	03A34004	sllv	$8,$v1,$29
00000218	03255807	srav	$t3,$a1,$t9
0000021c	3C1F4D0F	lui	$ra,19727
00000220	0329682B	sltu	$t5,$t9,$9
00000224	015F0019	multu	$t2,$31
00000228	340F38D6	ori	$15,$0,14550
0000022c	021FB824	and	$s7,$16,$31
00000230	0000B012	mflo	$s6
00000234	3C0A0873	lui	$10,2163
00000238	02C60019	multu	$22,$a2
0000023c	3C004A53	lui	$0,19027
00000240	2AAC74BD	slti	$t4,$s5,29885
00000244	01538821	addu	$s1,$10,$19
00000248	00086582	srl	$t4,$t0,22
0000024c	32451FE3	andi	$a1,$s2,8163
00000250	0012A0C0	sll	$20,$18,3
00000254	00065D00	sll	$11,$a2,20
00000258	00003812	mflo	$a3
0000025c	033B902B	sltu	$18,$t9,$27
00000260	004EA025	or	$20,$2,$t6
